feature-libs/user/profile/styles/my-account-v2/_my-account-v2-email.scss

```diff
@@ -0,0 +1,81 @@
+%cx-my-account-v2-email {
+  .flex-line {
+    display: flex;
+    flex-direction: row;
+    align-items: baseline;
+  }
+
+  .headertext {
+    @include type('3');
+    padding-bottom: 1.5rem;
+    width: 95%;
+  }
+
+  .editButton {
+    @include type();
+    color: var(--cx-color-primary);
+    border: none;
+    background-color: var(--cx-color-transparent);
+  }
+
+  .text-head {
+    @include type('6');
+    min-width: 7.5rem;
+    color: var(--cx-color-background-dark);
+    width: 15%;
+  }
+
+  .value {
+    @include type();
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+  }
+
+  .button {
+    @include type();
+    padding: 0.8rem, 1.5rem, 0.8rem, 1.5rem;
+  }
+
+  .button-cancel {
+    color: var(--cx-color-primary);
+  }
+
+  .email-editing-area {
+    width: 100%;
+    max-width: 35rem;
+  }
+
+  .cx-message-info {
+    @include type('7');
+    border-color: var(--cx-color-visual-focus);
+    background-color: var(--cx-color-info-accent);
+    .cx-message-icon {
+      cx-icon {
+        color: var(--cx-color-primary-accent);
+      }
+    }
+  }
+
+  .btn-group-row {
+    padding-top: 0.75rem;
+    display: flex;
+    flex: 100%;
+    justify-content: end;
+  }
+
+  .btn-group {
+    display: flex;
+    justify-content: space-between;
+    width: 40%;
+  }
+
+  form {
+    label {
+      padding-bottom: 0.75rem !important;
```

Comment:
We avoid using `!important`. Can you make the scss more specific so you won't need to use !important.